catalog/spatial/wfs/2.0.0/spatial-wfs-v2_0_0-source/pom.xml

```diff
@@ -177,6 +184,7 @@
                             spatial-wfs-converter;scope=compile|runtime,
                             spatial-wfs-v2_0_0-converter;scope=compile|runtime,
                             catalog-core-api-impl,
+                            catalog-core-api-impl;groupId=org.codice.alliance.catalog.core,
```

Comment:
I think this is pulling in two different catalog-core-api-impls. One from ddf (i'd guess) and the other from alliance. Because we have name clashes between alliance/ddf you have to specify group ID. 

Might help to clarify by explicitly adding the ddf group id to the other catalog-core-api-impl.